extension-config.json

```diff
@@ -1,17 +1,18 @@
 {
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
for the other extension that we migrated already we did a major version increase. We should do here the same

Reply:
Updated to version 2.0.0